keras_nlp/models/gpt2/gpt2_backbone.py

```diff
@@ -213,7 +214,7 @@ def create_layout_map(cls, mesh):
                 distribution, and the second for model parallel distribution.
 
         Returns:
-            A `tf.keras.dtensor.experimental.LayoutMap` which contains the
+            A `keras.dtensor.experimental.LayoutMap` which contains the
```

Comment:
Does this path make sense for Keras Core?

Reply:
It does not. Let's stick to `tf.keras` for clarity.